ArduPlane/takeoff.cpp

```diff
@@ -135,13 +135,17 @@ void Plane::takeoff_calc_roll(void)
     // the takeoff altitude
     float roll_limit = roll_limit_cd*0.01f;
     float baro_alt = barometer.get_altitude();
-    // below 5m use the LEVEL_ROLL_LIMIT
-    const float lim1 = 5;    
-    // at 15m allow for full roll
-    const float lim2 = 15;
+
+    // lim1 - below altitude LEVEL_ROLL_LIMIT restrict roll
+    // lim2 - above altitude (LEVEL_ROLL_LIMIT * 2) allow full roll.
```

Comment:
comment typo

Reply:
changed to 
    // lim1 - below altitude TKOFF_LVL_ALT, restrict roll to LEVEL_ROLL_LIMIT
    // lim2 - above altitude (TKOFF_LVL_ALT * 2) allow full flight envelope of LIM_ROLL_CD